Typo correction in FAQ and Corrected misinformation

hyeyeon919 opened the pull request:
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

While translating the app in Korean I was able to find a typo error in the FAQ.
I corrected storgea to storage.
Next is the location where the file is saved is currently changeable.
However, the said non-changeable. So I deleted the word. non-changealbe.

Thank you.


Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

codegsaini reviewed (changes requested):
Commit title is out of way from your proposed changes :(

codegsaini commented on app/src/main/res/values/strings.xml:
```diff
@@ -578,7 +579,7 @@
         <item>Can I read PDF files using the app?#####No, you cannot as of now. This app does not work as a PDF viewer.</item>
         <item>Where are the PDF files saved I created using the app?#####The PDF files files created with the app are automatically saved in the folder named PDFfiles on your device storage. \nYou can also change the default location in <strong>Setings</strong> section.</item>
-        <item>How to change the default storgae location?#####To change the default storage location, \n1) Click the <strong>Hamburger icon</strong> (three horizontal line) at the top-left corner. \n2) Scroll down and click <strong>Settings</strong>. \n3) In the settings page, click the field just beneath the text <strong>Change storage location</strong>. \n4) Open the preferred save location and ...
+        <item>How to change the default storage location?#####To change the default storage location, \n1) Click the <strong>Hamburger icon</strong> (three horizontal line) at the top-left corner. \n2) Scroll down and click <strong>Settings</strong>. \n3) In the settings page, click the field just beneath the text <strong>Change storage location</strong>. \n4) Open the preferred save location and ...
```
Good job, please mention this changes in pull request's description.

codegsaini commented on app/src/main/res/values/strings.xml:
```diff
@@ -469,7 +469,7 @@
     <string name="settings_info">You can modify the default value for various settings here.</string>
     <string name="storage_location_modified">Storage location changed</string>
-    <string name="storage_location_info">Storage location(non-changeable)</string>
+    <string name="storage_location_info">Storage location</string>
```
Good

codegsaini commented on app/src/main/res/values/strings.xml:
```diff
@@ -568,6 +568,7 @@
     <string name="snackbar_files_deleted">Files Deleted</string>
     <string name="lbl_recently_used">Recently Used Features</string>
 
+    
```
Unnecessary change :(

codegsaini commented on app/src/main/res/values-ko-rKR/arrays.xml:
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
+
```
Unnecessary change :(

codegsaini commented on app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java:
```diff
@@ -1,8 +1,6 @@
 
 import android.content.Context;
-import android.support.test.InstrumentationRegistry;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
```
Changes are different than what title is telling about.

hyeyeon919 replied:
Thank you for letting me know about the problems. I'm sorry I am not a good user of GitHub :(
I closed my PRs add oped new ones. And I fixed everything you said. 
I hope you check it out soon :)
Thank you for your feedback!

codegsaini reviewed (changes requested):
Please add proper description in pull request description about proposed changes and also do some changes mentioned with individual comment.

hyeyeon919 commented:
Hi 
I fixed the problem you commented.
Please check my new PRs.
And it would be nice if you merge it !!
Thank You! :)
-----Original Message-----
From: "Gaurav ***@***.***>
To: ***@***.***>;
Cc: "hyeyeon ***@***.***>; ***@***.***>;
Sent: 2021-12-15 (수) 22:52:16 (GMT+09:00)
Subject: Re: [Swati4star/Images-to-PDF] Typo correction in FAQ and Corrected misinformation (PR #1040)
 
@codegsaini requested changes on this pull request.
Please add proper description in pull request description about proposed changes and also do some changes mentioned with individual comment.
In app/src/main/res/values/strings.xml:
> @@ -578,7 +579,7 @@ <item>Can I insert captions in the image after adding?#####No, captions cannot be inserted on images except otherwise edited on the image itself.</item> <item>Can I read PDF files using the app?#####No, you cannot as of now. This app does not work as a PDF viewer.</item> <item>Where are the PDF files saved I created using the app?#####The PDF files files created with the app are automatically saved in the folder named PDFfiles on your device storage. \nYou can also change the default location in <strong>Setings</strong> section.</item> - <item>How to change the default storgae location?#####To change the default storage location, \n1) Click the <strong>Hamburger icon</strong> (three horizontal line) at the top-left corner. \n2) Scroll down and click <strong>Settings</strong>. \n3) In the settings page, click the field just beneath the text <strong>Change storage location</strong>. \n4) Open the preferred save location and click the <strong>Select icon</strong> at the top. \nOnce changed, the files created using the app will be stored in the new location.</item> + <item>How to change the default storage location?#####To change the default storage location, \n1) Click the <strong>Hamburger icon</strong> (three horizontal line) at the top-left corner. \n2) Scroll down and click <strong>Settings</strong>. \n3) In the settings page, click the field just beneath the text <strong>Change storage location</strong>. \n4) Open the preferred save location and click the <strong>Select icon</strong> at the top. \nOnce changed, the files created using the app will be stored in the new location.</item>
Good job, please mention this changes in pull request's description.
In app/src/main/res/values/strings.xml:
> @@ -469,7 +469,7 @@ <string name="font_family_value_def">Font family default value : %s </string> <string name="settings_info">You can modify the default value for various settings here.</string> <string name="storage_location_modified">Storage location changed</string> - <string name="storage_location_info">Storage location(non-changeable)</string> + <string name="storage_location_info">Storage location</string>
Good
In app/src/main/res/values/strings.xml:
> @@ -568,6 +568,7 @@ <string name="snackbar_files_deleted">Files Deleted</string> <string name="lbl_recently_used">Recently Used Features</string> +
Unnecessary change :(
In app/src/main/res/values-ko-rKR/arrays.xml:
> @@ -1,4 +1,5 @@ <?xml version="1.0" encoding="utf-8"?> +
Unnecessary change :(
In app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java:
> @@ -1,8 +1,6 @@ package swati4star.createpdf; import android.content.Context; -import android.support.test.InstrumentationRegistry; -import androidx.test.ext.junit.runners.AndroidJUnit4;
Changes are different than what title is telling about.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
